conf/interpreter-list

```diff
@@ -29,6 +29,7 @@ hbase           org.apache.zeppelin:zeppelin-hbase:0.7.0                Hbase in
 ignite          org.apache.zeppelin:zeppelin-ignite_2.11:0.7.0          Ignite interpreter built with Scala 2.11
 jdbc            org.apache.zeppelin:zeppelin-jdbc:0.7.0                 Jdbc interpreter
 kylin           org.apache.zeppelin:zeppelin-kylin:0.7.0                Kylin interpreter
+java            org.apache.zeppelin:zeppelin-java:0.7.0                 Java interpreter
```

Comment:
The version here seems outdated, I created https://issues.apache.org/jira/browse/ZEPPELIN-3660 for it